package.json

```diff
@@ -28,6 +28,7 @@
     "@babel/preset-env": "^7.11.0",
     "@babel/register": "^7.10.5",
     "@comicrelief/eslint-config": "^1.2.0",
+    "babel-eslint": "^10.1.0",
```

Comment:
`babel-eslint` is deprecated and we have the correct `@babel/eslint-parser` package installed. The ESLint error was because of an issue with the `eslint-plugin-flowtype` package. Not sure why this started failing now, given linting passed on `master`. You can work around it by setting the parser explicitly in `.eslintrc.yml`:

```yml
parser: '@babel/eslint-parser'
```

Reply:
Ah, I thought you might have some insight on this! I guessed it was something odd in my configuration but since I couldn't work out what it was, the quickest solution seemed to be adding the dev dependency for `babel-eslint`. I'll reconfigure using the edit of `.eslintrc.yml` as you suggest instead.